content/en/guides/outbound-oss.md

```diff
@@ -0,0 +1,724 @@
+
+The main difference of a DCO compared to a CLA is, that a DCO is not a contract, it is a kind of attest of the specific contributor that they are entitled to submit a concrete contribution. All the effort which has to be spent to get a CLA signed and maintained is not needed. The only tasks which have to be carried out are:
+
+* Evaluation of the DCO by the "Legal department"
+* Evaluation by the "IP department"
+* Evaluation by the specific contributor, whether it is acceptable for them
+
+Since the DCO version 1.1 is the "standard" the "Legal"- and "IP department" only have very little effort to spend.
+
+## Procedure for contributions to existing projects
+
+The more complex the business environment in which the code to publish was developed, the more stakeholders need to be involved. The picture below shows a procedure that involves all functions, even in a complex setup.
+
+![contributions](../img/template-contribs.png)
+
+Abbreviations used:
+
+* CLA = Contributor License Agreement
+* DCO = Developers Certificate of Origin
+* ECC = Export Control and Customs
+* IP  = Intellectual Property
+
+The procedure shown above is not suited for frequent contributors and/or contributors who are working “upstream” in their daily work. For these developers different procedures need to be established in order to avoid loading them with “unproductive” work. Different contribution models can be established in an organization to serve different needs.
+
+### Contribution models
+
+The following approaches are suited for such developers:
+
+* Small contributions model
+* Major to major release model
+* Full trust model
+
+![small-contributions](../img/small-contributions.png)
+
+#### Small contributions model or trivial contributions
+
+A small or trivial contribution is a rather small and simple change to already existing open source software. Typical cases found in this category are bug fixes with no or low Intellectual Property value.
+
+A change is not trivial if:
+
+* Functionality is added or changed.
+* The interface of the open source software component is changed.
+* It is an optimization that more than insignificantly increases performance.
+* It contains a design or an algorithm that wouldn’t be obvious for a software engineer.
+
+This procedure scopes small contributions. It can be implemented for small or trivial contributions following the initial contribution to a particular OSS project or component. The initial contribution has to undergo the entire procedure described above, because CLAs/DCOs etc. have to be checked  and signed in case the particular project requires them.
+After the initial contribution all subsequent small contributions can be contributed directly to the OSS project without the need to follow the defined process no matter which version of the OSS project.
+
+#### Major to major release model
```

Comment:
It would be great if the text could more explicitly explain the motivation for using releases as decision points for approvals. These decision points are typically not closely related to the properties checked, such IP, CLA/DCO contents, i.e., those things do not typically change across releases. So, is this idea behind tying re-approval to releases primarily to have some form of regular check?

Reply:
Discussion continues in https://github.com/todogroup/outbound-oss/issues/64